src/requirements.txt

```diff
@@ -37,7 +53,13 @@ python-jose==2.0.2
 pytz==2017.2
 PyYAML==3.12
 requests==2.18.4
+simplegeneric==0.8.1
 six==1.10.0
 SQLAlchemy==1.1.14
+traitlets==4.3.2
 urllib3==1.22
+vine==1.1.4
+virtualenv==16.0.0
+virtualenv-clone==0.3.0
+wcwidth==0.1.7
```

Comment:
Some dependencies have been introduced that seems to be specific for development, you can chuck them out, or use `pipenv install <package> --dev> to make sure we don't pollute our dependencies with unnecessary packages.